internal/api/front/perses/src/app/project/dashboard/dashboardfeed.model.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 Amadeus s.a.s
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export interface Serie {
+  metric: Record<string, string>;
+  values: [number, string][];
+}
+
+export interface Result {
+  result: Serie[];
+}
+
+export interface Panel {
+  name: string;
+  order: number;
+  results: Result[];
+}
+
+export interface DashboardFeedModel {
```

Comment:
I would called it more `SectionFeedModel`. First because it fits the name of the struct on the backend side: https://github.com/perses/perses/blob/93a0c7961d682e78a691f83c51561257eb21afaa/pkg/model/api/v1/dashboard_feed.go#L34-L42

Then also because the DashboardFeedModel if it would exist, it would contain as well the value of the variable. Which is not the case, since it is separated in another struct.

It is separated into two different struct, because the changes of the variable would require then recalcule the data of the different panel. But the reverse way doesn't exist. So since it's two differents events, you have then two differents endpoints and struct associated.